extensions/2.0/Khronos/KHR_image_formation/README.md

```diff
@@ -0,0 +1,197 @@
+<!--
+Copyright 2018-2020 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR\_image\_formation
+
+## Contributors
+
+* Romain Guy, Google [@romainguy](https://twitter.com/romainguy)
+* Emmett Lalish, Google [@emmettlalish](https://twitter.com/emmettlalish)
+
+Copyright 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+The scene referred values computed during the lighting pass using the glTF 2.0 spec are luminance values, expressed in cd.m-2 or nits. It follows that light incident to our virtual optical system can cover a large range of values when physical light units (KHR_lights_punctual) are used, from 10-5 cd.m-2 for starlight to 109cd.m-2 for the sun. However the common methods we have of storing and displaying images have limited dynamic range. Of course this problem does not just apply to rendering; the human eye also has limited dynamic range, and of course photographs printed on paper (or displayed on a screen) cannot possibly reproduce the range  of intensity that was incident on the lens. 
+
+This extension is intended to allow the producer of a glTF to define all the parameters necessary for a consumer (renderer) to generate a consistent output image of the scene, given consistent lighting and view. Consistent does not imply pixel-identical, as the rendering is still subject to many arbitrary approximations, but simply that the output would be identical if the physics were respected exactly. What this extension defines is a function that maps [0, infinity) input light RGB to the [0, 1] output encoding color gamut (most commonly sRGB). In practice, this function has a much larger effect on the appearance of the output than the differences in renderer approximations.
+
+Supporting this extension is only appropriate for rendering a single glTF scene; since it defines a global output mapping it is impossible to respect conflicting mappings for two glTFs in the same scene. Likewise, if a shop wanted a consistent “look” across  their portfolio, they might be better served by picking a consistent mapping function and requiring their artists to design with this mapping in mind, rather than displaying different products with unique parameters. 
```

Comment:
It would seem that this prohibits the usecase of mixing models, for instance a 'Metaverse' type of usecase where models from a number of different suppliers are displayed within the same scene - is this correct?

Reply:
Exactly; global configuration like environment lighting, camera, and tone-mapping are necessarily scene-level, so if the app has a scene composed of many glTFs, then it doesn't make sense to support this extension. This extension is more useful for the single-model cases like Commerce where you want to completely specify the sRGB output. Granted, environment lighting also needs to be consistent as stated above.

Reply:
> Exactly; global configuration like environment lighting, camera, and tone-mapping are necessarily scene-level, so if the app has a scene composed of many glTFs, then it doesn't make sense to support this extension.

So we are in agreement that this extension does not solve the case of viewing multiple models or viewing a model under varying light conditions - for instance an AR/Metaverse usecase that mixes models, or any usecase that alters the light setup?

Reply:
> This extension is more useful for the single-model cases like Commerce where you want to completely specify the sRGB output.

Is it safe to assume that you are referring to a MPP (main product picture) usecase?

Reply:
Yes, I'm concerned with 3D replacing the main product picture (this is the main use case our partners are working with us on) and not the metaverse or AR, where the lighting and post-processing are app-defined or camera-defined already.

Reply:
It seems to me that for your usecase you see the lighting being declared in the asset - is this the case?
In what scenario do you see use for the 'exposure' parameter?